### PR TITLE
STCOM-785: interactors update: text-field

### PR DESCRIPTION
**Pull request by jgnieuwhof**
## Motivation
Upgrade the `TextField` interactor to the new bigtest interactor syntax, and make it available for consumption by the test suite.

## Approach
Added a `TextField` interactor, with filters and actions that cover the common ways of interacting with a text field.

See this in action here: https://github.com/folio-org/stripes-components/pull/1468

### CODE REVIEW

File: interactors/text-field.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export default createInteractor('text field')({
+  selector: 'div[class^=textField-]',
+  locator: label,
+  filters: {
+    id: (el) => el.querySelector('input').id,
+    label,
+    type: (el) => el.querySelector('input').type,
+    value: (el) => el.querySelector('input').value,
+    focused: (el) => el.querySelector('input').contains(el.ownerDocument.activeElement),
+    readOnly: (el) => el.querySelector('input').hasAttribute('readOnly'),
```
Comment (jbolda) [inline interactors/text-field.js:19]: /cc @elrickvm, maybe this is better than the double bang we were doing last night.

File: interactors/index.js [hunk 2]
```diff
@@ -10,5 +10,4 @@ export { default as Select } from './select';
 export { default as Label } from './label';
 export { default as Button } from './button';
 export { default as Pane } from './pane';
-
-export { TextField } from '@bigtest/interactor';
```
Comment (jbolda) [inline interactors/index.js:14]: Do we need to adjust the tests that relied on this lower level version of `TextField`?

Comment (jgnieuwhof) [reply interactors/index.js:14]: I don't think so 🤔. I did some grepping, couldn't find this in-use, and the test suite is passing locally.

Comment (jbolda) [reply interactors/index.js:14]: Looks like I might have added it, switched to a different direction and then never removed it? Appears that we don't need to do anything with it.

**Review by jbolda — APPROVED** [review]

File: interactors/text-field.js (hunk 1 above)
Comment (elrickvm) [reply interactors/text-field.js:19]: nice!

**Review by elrickvm — APPROVED** [review]

**Review by cowboyd — APPROVED** [review]

**Pull request merged**